source/b_usermanagement.rst

```diff
@@ -1,3 +1,38 @@
+.. _AuthenticationAnchor:
+
+==========
+Signing in
+==========
+
+First-time users require an invitation to create a Lizard account. An
+invitation can be sent by someone with a 'manager' role in your organisation.
+If you do not know whom to contact, please contact our support office
+(servicedesk@nelen-schuurmans.nl).
+
+Users that have used Lizard before can click the "Log in" button in the top
+right of Lizard.
+
+After clicking "Log in" or after following the invitation link, you will be
+redirected to the domain "auth.lizard.net/login?...".
```

Comment:
Maybe we can make it so that the word links to the domain. Its a bit cleaner (.. you will redirected to the [log in](url))